docs/docs/frigate/installation.md

```diff
@@ -132,6 +132,71 @@ If you are using `docker run`, add this option to your command `--device /dev/ha
 
 Finally, configure [hardware object detection](/configuration/object_detectors#hailo-8l) to complete the setup.
 
+### MemryX MX3  
```

Comment:
This should go lower under the community supported section for now. You can also add it to the info card at the top to link to it